jun.js

```diff
@@ -38,7 +38,7 @@ function setDemo(e) {
     "                file gas.log size 10k files 3;\n                flag all;\n            }\n        }\n    }\n}";
 }
 
-function FiltrInactive(leaf) {
+function filtrInactive(leaf) {
```

Comment:
ok, but you change only name of func, and forget change  every calls of this func